compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -851,6 +877,29 @@ TEST(QuantizedModelVerifierTest, LocalCreateDummyConst)
   EXPECT_NO_THROW(create_dummy_const<Type::FLOAT32>(&g, {32, 32}));
 }
 
+TEST(QuantizedModelVerifierTest, LocalCreateConst)
+{
+  loco::Graph g;
+  std::initializer_list<float> values = {0.1, 0, -5, 100};
+  luci::CircleConst *node = create_const<Type::FLOAT32, float>(&g, {2, 2}, values);
+
+  uint32_t index = 0;
+  for (auto val : values)
+  {
+    EXPECT_EQ(node->at<Type::FLOAT32>(index++), val);
+  }
+}
+
+TEST(QuantizedModelVerifierTest, LocalCreateConst_NEG)
+{
+  loco::Graph g;
+  std::initializer_list<float> values = {0.1, 0, -5, 100};
+  luci::CircleConst *node = create_const<Type::FLOAT32, float>(&g, {2, 2}, values);
+
+  // access out of bound
+  EXPECT_DEATH(node->at<Type::FLOAT32>(4), "Assertion `n < size<DT>()");
```

Comment:
I guess this don't work with release test, because assert doens't work.

Reply:
yes, assert check is a bit difficult with gtest for this reason.